hack/make-rules/test-cmd-util.sh

```diff
@@ -4669,3 +4669,257 @@ runTests() {
     exit 1
   fi
 }
+
+run_initializer_tests() {
+  set -o nounset
+  set -o errexit
+
+  create_and_use_new_namespace
+  kube::log::status "Testing --include-uninitialized"
+  ### Create the Initializer
+  kubectl create -f hack/testdata/initializer-configuration.yaml 2>&1 "${kube_flags[@]}"
+  kube::test::get_object_assert initializerconfiguration "{{range.items}}{{$id_field}}{{end}}" 'test-initializer'
+
+  ### Create a deployment
+  kubectl create -f hack/testdata/initializer-deployments.yaml 2>&1 "${kube_flags[@]}" || true
+
+  ### Test kubectl get --include-uninitialized
+  # Command
+  output_message=$(kubectl get deployments 2>&1 "${kube_flags[@]}")
+  # Post-condition: The text "No resources found" should be part of the output
+  kube::test::if_has_string "${output_message}" 'No resources found'
+  # Command
+  output_message=$(kubectl get deployments --include-uninitialized=false 2>&1 "${kube_flags[@]}")
+  # Post-condition: The text "No resources found" should be part of the output
+  kube::test::if_has_string "${output_message}" 'No resources found'
+  # Command
+  output_message=$(kubectl get deployments --include-uninitialized 2>&1 "${kube_flags[@]}")
+  # Post-condition: The text "No resources found" should not be part of the output
+  kube::test::if_has_not_string "${output_message}" 'No resources found'
```

Comment:
can you check that deployment's name appears in the output instead of this? this makes the test more readable, too, because otherwise we have to read `if_has_not_string` and `if_has_string` every time.